lapce-ui/src/status.rs

```diff
@@ -482,6 +482,23 @@ impl Widget<LapceTabData> for LapceStatus {
 
         let mut right = size.width - 5.0;
         if let Some(editor) = &data.main_split.active_editor() {
+            // Do not draw file language and cursor/text details for non-file tabs
+            if let Some(editor_tab_id) = editor.tab_id {
+                if let Some(editor_tab) =
+                    data.main_split.editor_tabs.get(&editor_tab_id)
+                {
+                    if let Some(child) = editor_tab.active_child() {
+                        match child {
+                            EditorTabChild::Settings { .. }
+                            | EditorTabChild::Plugin { .. } => {
+                                return;
```

Comment:
This could cause a surprise if we'll have something to paint after this.

Reply:
I meant we should probably extract this paint in a new fn and return there.

Reply:
@dzhou121 does that look better?

Reply:
Might as well split document language name and details into own functions, but I'm lazy :P